spec/features/profile/view_profile_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+RSpec.describe 'View Profile', type: :feature do
+  let(:log_in_page) { LogInPage.new }
+  let(:view_profile_page) { ViewProfilePage.new }
+  let(:edit_profile_page) { EditProfilePage.new }
+  let(:student) { create(:student, courses: [course1, course2, course3]) }
```

Comment:
Generating data for a has_many https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations